src/ucp/core/ucp_worker.c

```diff
@@ -578,7 +578,7 @@ void ucp_worker_iface_activate(ucp_worker_iface_t *wiface, unsigned uct_flags)
 {
     ucp_worker_h worker = wiface->worker;
 
-    ucs_trace("activate " UCP_WIFACE_FMT " acount=%u aifaces=%u",
```

Comment:
This is not a typo. `acount` - `activate_count`. Please revert the change, or replace by `a_count`, or `activate_count`, or `activation counter`.

Reply:
Nice catch, thanks, fixed